pennylane/ops/qubit/observables.py

```diff
@@ -197,6 +197,18 @@ class SparseHamiltonian(Observable):
         do_queue (bool): Indicates whether the operator should be
             immediately pushed into the Operator queue (optional)
         id (str or None): String representing the operation (optional)
+
+    **Example:**
+        You can construct your Hamiltonian as usual using :class:`~.Hamiltonian` and then utilize
+        the utility function :func:`~.utils.sparse_hamiltonian` to construct the sparse matrix that serves as the input
+        to ``SparseHamiltonian``:
+
+        >>> wires = 20
+        >>> coeffs = [1 for _ in range(wires)]
+        >>> observables = [qml.PauliZ(0) for _ in range(wires)]
+        >>> H = qml.Hamiltonian(coeffs, observables)
+        >>> Hmat = qml.utils.sparse_hamiltonian(H)
+        >>> H_sparse = qml.SparseHamiltonian(Hmat, wires=wires)
```

Comment:
I think we don't want this to be indented, it can have implications for the rendering!